subcommittees/trainers/minutes/2017-03-23-business.md

```diff
@@ -0,0 +1,93 @@
+# Carpentry Instructor Trainers Group
+
+## Trainer Business Meeting  (Note time change outside of USA)
+- Thursday, March 23, 2017 UTC 22:00 (Erin)
+- Brisbane 8am FRIDAY, Melbourne 9am FRIDAY, Auckland 11am FRIDAY, San Francisco 3pm THURSDAY, Philadelphia 6pm THURSDAY
+- See this link for your local time: https://www.timeanddate.com/worldclock/fixedtime.html?iso=20170323T15&p1=224
+
+## Attending UTC 14:00 (Maneesha)  x = present
+- Maneesha Sane x
+- Aleks Nenadic x
+- Kari L. Jordan x
+- Jeff Oliver x
+- Rayna Harris x
+- Mateusz Kuzak x
+- Chris Hamm x
+- Christina Koch x
+
+## Attending UTC 22:00 (Erin)
+- Christina Koch x
+- Erin Becker x
+- Belinda Weaver x
+- Anelda van der Walt x
+- Ariel Rokem x
+- Naupaka Zimmerman x
+- Karen Word x
+
+## Apologies
+- Steve Crouch
+
+
+## Agenda
+
+## Introductions
+
+## Announcement about curriculum (Christina)
+- official maintiners. in the past it was Christina and Greg. Now its Christina and Erin. 
+- working on resolving old issues and cleaning the docket. new developments will be discussed at these meetings
+- quick fix - easy things that can be resolved with a pull request without further discussion
+- Erin and Cristina will give us an update on all the tags once they finalize the meaning
+- specific question from Anelda: with upcoming workshop, can shorten carpentries module (based on upcoming potential merger?)  Answer: should still cover same material, but if it takes less than an hour, that's okay.  
+
+## Scheduling training events (update for June-August)
+- new process, organizing schedule around trainer's calendars.  Saying "trainers are available at these dates" and have the partner sites sign up. 
+- everyone reported their availability here https://docs.google.com/spreadsheets/u/1/d/10qr987YT9qYi0NGO1sJhe3OHkIL70bbqtUbJNvhfC58/edit
+- Meesha has put together a tentative schedule in the same document above
+- Everyone has been assigned a partner and a few dates to choose from
+- note: Belinda is running a training in June that's not listed (June 19-20, Sydney)
+- workshops that have been pre-arranged are not necessarily on this calendar
+- **ACTION ITEM**: everyone discuss with your partner and chose a date. Let Maneesha know by Thursday March 30th. Then those times will be "confirmed." 
+- These dates will then be shared with the partner sites and they will be asked to sign up at least one month in advance. 
+- This seems to be working for people 
+- If you didn't sign up for a time in this time range, shadow an event in your time zone
+- Question: Are these online or in person: Answer. These will all be online training event. 
+- Moving forward, we will continue this process four times a year so that we always have a plan three months out. For non-summer months, the goal will be to offer 1 time zone per month.
```

Comment:
Does this mean that we are setting up 1 training per month? Or is it one training per timezone per month (that would be 6 trainings per month).